includes/templates/class-amp-post-template.php

```diff
@@ -260,11 +170,11 @@ private function build_post_content() {
 
 		$this->add_data_by_key( 'post_amp_content', $amp_content->get_amp_content() );
 		$this->merge_data_for_key( 'amp_component_scripts', $amp_content->get_amp_scripts() );
-		$this->merge_data_for_key( 'post_amp_styles', $amp_content->get_amp_styles() );
+		$this->merge_data_for_key( 'amp_styles', $amp_content->get_amp_styles() );
```

Comment:
We probably shouldn't change this for back-compat reasons. Or we should find a workaround that gracefully handles the change.